extensions/opentelemetry/runtime/src/main/java/io/quarkus/opentelemetry/runtime/graal/Substitutions.java

```diff
@@ -0,0 +1,58 @@
+package io.quarkus.opentelemetry.runtime.graal;
+
+import java.io.Closeable;
+import java.util.Collections;
+import java.util.List;
+import java.util.function.BiFunction;
+
+import com.oracle.svm.core.annotate.Substitute;
+import com.oracle.svm.core.annotate.TargetClass;
+
+import io.opentelemetry.api.events.EventEmitterProvider;
+import io.opentelemetry.api.metrics.MeterProvider;
+import io.opentelemetry.sdk.autoconfigure.spi.ConfigProperties;
+import io.opentelemetry.sdk.logs.SdkLoggerProviderBuilder;
+import io.opentelemetry.sdk.logs.export.LogRecordExporter;
+import io.opentelemetry.sdk.metrics.export.MetricExporter;
+import io.opentelemetry.sdk.metrics.export.MetricReader;
+
+public class Substitutions {
+
+    @TargetClass(className = "io.opentelemetry.sdk.autoconfigure.MeterProviderConfiguration")
+    static final class Target_MeterProviderConfiguration {
+
+        @Substitute
+        static List<MetricReader> configureMetricReaders(
+                ConfigProperties config,
+                ClassLoader serviceClassLoader,
+                BiFunction<? super MetricExporter, ConfigProperties, ? extends MetricExporter> metricExporterCustomizer,
+                List<Closeable> closeables) {
+            // OTel metrics not supported and there is no need to call
```

Comment:
does it mean that we cannot use these configuration to enable Prometheus exporter:
https://github.com/open-telemetry/opentelemetry-java/blob/main/sdk-extensions/autoconfigure/README.md#prometheus-exporter

Reply:
Correct. No OTel Metrics on Quarkus for now. It requires a lot of rework in the instrumentation.